PROJECT

```diff
@@ -27,6 +27,10 @@ resources:
     defaulting: true
     validation: true
     webhookVersion: v1
+- controller: true
```

Comment:
Why do we now have a secret controller?

Reply:
Weird leftover from first iteration, will remove